sdk/storage/azure-storage-file-datalake/tests/test_dictmixin.py

```diff
@@ -0,0 +1,76 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+
+from azure.storage.filedatalake._models import DictMixin
+
+
+class TestDictMixin:
```

Comment:
Since this is shared code (in our `_shared` folders) it probably does not need to be tested in every package. You could remove the test files for every package and leave only Blob just to reduce copy/pasted code.